reya_trading/config.py

```diff
@@ -0,0 +1,72 @@
+"""
+Configuration settings for the Reya Trading API.
+"""
+import os
+from dataclasses import dataclass
+from typing import Dict, Optional, Any
+from dotenv import load_dotenv
+
+MAINNET_CHAIN_ID = 1729
+
+@dataclass
+class TradingConfig:
+    """Configuration for Reya Trading API"""
+    api_url: str
+    chain_id: int
+    private_key: Optional[str] = None
+    orders_gateway_address: Optional[str] = None
+    account_id: Optional[int] = None
+    wallet_address: Optional[str] = None
+    
+    @property
+    def is_mainnet(self) -> bool:
+        """Determine if current chain ID is mainnet"""
+        return self.chain_id == MAINNET_CHAIN_ID
+    
+    @property
+    def default_orders_gateway_address(self) -> str:
+        """Get default OrdersGateway proxy contract address based on chain ID"""
+        if self.is_mainnet:
+            return "0xfc8c96be87da63cecddbf54abfa7b13ee8044739"  # Mainnet address
+        else:
+            return "0x5a0ac2f89e0bdeafc5c549e354842210a3e87ca5"  # Testnet address
+    
+    @property
+    def default_conditional_orders_address(self) -> str:
+        """Get default ConditionalOrders proxy contract address based on chain ID"""
```

Comment:
there's no conditional orders proxy contract, is this referenced somewhere in ts sdk as well?